docs/provisioners/oci_artifacts.md

```diff
@@ -0,0 +1,141 @@
+# Plain Provisioner
+
+## Summary
+
+The `oci_artifacts` provisioner is a RukPak [provisioner](https://github.com/operator-framework/rukpak/tree/main/internal/provisioner) that knows how to interact with [OCI Artifact](https://github.com/opencontainers/artifacts#project-introduction-and-scope) images that contain operator bundle manifests.
+
+These `oci_artifact` bundles are container images containing a set of static Kubernetes YAML manifests that define an operator deployment. For more information on the `oci_artifact` format, see
+the [oci artifact bundle definition doc](/docs/bundles/oci_artifacts.md).
+
+The `oci_artifact` provisioner is able to unpack a given `oci_artifact` bundle onto a cluster and then instantiate it, making the content of the bundle available on-cluster. It does so by reconciling `Bundle` types that have the `spec.provisionerClassName` field set to `core-rukpak-io-oci-artifacts`. This field must be set to the correct provisioner
+name in order for the `oci_artifacts` provisioner to see and interact with the bundle.
+
+### Install and apply a specific version of an operator shippied as OCI Artifacts
+
+> :warning: Anyone with the ability to create or update BundleDeployment objects can become cluster admin. It's important
+> to limit access to this API via RBAC to only those that explicitly require access, as well as audit your bundles to
+> ensure the content being installed on-cluster is as-expected and secure.
+
+The `oci_artifact` provisioner can make available a specific `OCI Artifact` bundle in the cluster.
+
+Simply create a `BundleDeployment` resource that contains the desired specification of a Bundle resource.
+The `oci-artifacts` provisioner will unpack the provided Bundle onto the cluster, and another `BundleDeployment` provisioner, eg the `plain` provisioner can eventually make the content available on the cluster.
+
+```yaml
+apiVersion: core.rukpak.io/v1alpha1
+kind: BundleDeployment
+metadata:
+  name: my-bundle-deployment
+spec:
+  provisionerClassName: core-rukpak-io-plain
+  template:
+    metadata:
+      labels:
+        app: my-bundle
+    spec:
+      source:
+        type: image
+        image:
+          ref: my-bundle@sha256:xyz123
+      provisionerClassName: core-rukpak-io-oci-artifacts
```

Comment:
reiterating my comment above, i would not have expected a new provisioner type to be introduced for a new bundle source type.